org.eclipse.winery.yaml.common/src/main/java/org/eclipse/winery/yaml/common/reader/yaml/Builder.java

```diff
@@ -218,7 +228,9 @@ public String buildDescription(Object object) {
         Map<String, Object> map = (Map<String, Object>) object;
         return map.entrySet().stream()
```

Comment:
Wouldn't it be a more readble approach to extract this stream into a private method?
It also would reduce the amout of code clones in your code.